packages/ui/src/services/json-schema-document.service.ts

```diff
@@ -26,12 +27,12 @@ export class JsonSchemaDocument extends BaseDocument {
     super(documentType, documentId);
     this.name = documentId;
     const field = JsonSchemaDocumentService.createFieldFromJSONSchema(this, '', this.jsonSchema);
-    this.fields.push(field);
-    this.schemaType = 'JSON';
+    field.type !== Types.AnyType && this.fields.push(field);
```

Comment:
[nitpick] What do you think of using a traditional `if` statement instead of this short-circuit statement? I find it easier to reason about.

```suggestion
    if (field.type !== Types.AnyType) {
      this.fields.push(field);
    }
```

Alternatively, what about the shorter version?
```suggestion
    if (field.type !== Types.AnyType) this.fields.push(field);
```

still confusing, but a bit less :smiley: